app/Repositories/UserRepository.php

```diff
@@ -24,7 +25,9 @@ public function register(UserEntity $userEntity)
 
     public function find($user_id)
     {
-        $user = $this->user->find($user_id);
+        $user = Cache::remember('find_tweet', 60 * 60 * 24, function () use ($user_id) {
```

Comment:
why caching key name is `find_tweet`? 

also, what will happen if I tried to `find(1)` and `find(2)`? will it return the same user?